src/Conference/UpdateConference.jsx

```diff
@@ -0,0 +1,153 @@
+import React, { useState, useEffect } from "react";
+import Input from "../Input";
+import styled from "styled-components";
+import { useHistory, useParams } from "react-router-dom";
+import { updateConferenceById, getConferenceById } from "../api";
+
+const StyledConferenceForm = styled.div`
+  height: 400px;
+  padding: 20px;
+`;
+
+const StyledCardHeading = styled.h3`
+  background: #dbd8db;
+  width: 100%;
+  height: 55px;
+  font-size: 30px;
+  font-weight: bold;
+  text-align: center;
+  vertical-align: middle;
+`;
+
+const StyledCard = styled.div`
+  border-style: solid;
+  border-color: black;
+  border-radius: 25px;
+  background: #dbd8db;
+  width: 65%;
+  height: 370px;
+  transition: 0.3s;
+  :hover {
+    box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+  }
+  vertical-align: middle;
+  margin: 0 auto;
+`;
+
+const UpdateForm = () => {
+  let history = useHistory();
```

Comment:
why let here?

Reply:
We do a history.push later and assumed that for this to work history would need to be a let so it could be changed, is this not the case?

Reply:
As we want to keep the same history object throughout I don't think this needs to be `let` as we are not reassigning the variable `history`, history.push is just using a method on the history object.